assets/src/blocks/ENForm/FormGenerator.js

```diff
@@ -37,187 +37,184 @@ const Input = (props) => {
     },
     dependent_field,
     errors,
-    is_side_style
+    is_side_style,
   } = props;
 
-  return ((field) => {
-    switch (field?.input_type) {
-      case 'text':
-      case 'email':
-        return <TextInput {...{field, onInputChange, onBlur, errors, is_side_style}} />;
-      case 'checkbox':
-        return <CheckboxInput {...{field, onInputChange, onBlur, index, dependent_field, errors, is_side_style}} />;
-      case 'radio':
-        return <RadioInput {...{field, onInputChange, onBlur, errors, is_side_style}} />;
-      case 'country':
-        return <CountryInput {...{field, onInputChange, onBlur, errors, is_side_style}} />;
-      case 'position':
-        return <PositionInput {...{field, onInputChange, onBlur, errors, is_side_style}} />;
-      case 'hidden':
-        return <HiddenInput {...{field}} />;
-      default:
-        console.error(`Input type <${field?.input_type}> unknown.`, field);
-        return null;
+  return ( ( returnField ) => {
```

Comment:
This change causes the block to crash (and it's probably why the acceptance tests are failing 😬) I'm not sure why we use a function here instead of directly using a `switch` statement 🤔 I think @lithrel wrote this so maybe he can help with the refactoring! But anyway if we want to keep this change to `returnField` then I think that in the following line it should be `switch ( returnField?.input_type )` and then the props should for all inputs should be written like
```
{ ...{ field: returnField, onInputChange, onBlur, errors, is_side_style } }
```